manim/utils/space_ops.py

```diff
@@ -318,8 +319,10 @@ def angle_of_vector(vector: Sequence[float] | np.ndarray) -> float:
         c_vec = np.empty(vector.shape[1], dtype=np.complex128)
         c_vec.real = vector[0]
         c_vec.imag = vector[1]
-        return np.angle(c_vec)
-    return np.angle(complex(*vector[:2]))
+        val1: float = np.angle(c_vec)
+        return val1
+    val: float = np.angle(complex(*vector[:2]))
+    return val
 
 
 def angle_between_vectors(v1: np.ndarray, v2: np.ndarray) -> float:
```

Comment:
```suggestion
def angle_between_vectors(v1: Vector3DLike, v2: Vector3DLike) -> float:
```

Reply:
Same as earlier: as `VectorNDLike` aliases do not exist yet, you could use `VectorND` instead. I used `Vector3DLike` before, but this actually also works for N dimensions.
```suggestion
def angle_between_vectors(v1: VectorND, v2: VectorND) -> float:
```